src/sourdough-potato-bread.md

```diff
@@ -0,0 +1,32 @@
+# Potato sourdough bread
+
+![Potato sourdough bread](pix/sourdough-potato-bread.webp)
+
+- ⏲️ Prep time: 15 min
+- 🍳 Cook time: 45 min
+
+## Ingredients
+
+- 150g boiled potatoes in there skin
+- 350g wheat flour
+- 50g whole rye flour
+- 11g salt
```

Comment:
Typically we don't have salt in the ingredients list either... Usually I think that is for 'taste' at the end. But this is being baked in and has a specific measurement, so maybe this is fine to leave.

Reply:
I hope that it is okay in this case as you described. In this case the amount of salt is quite important. Also this way I can write `mix all ingredients together` without mention the amount of salt in the directions section.